kymatio/scattering1d/frontend/base_frontend.py

```diff
@@ -175,12 +175,11 @@ def meta(self):
                 The tuples indexing the corresponding scattering coefficient
                 in the non-vectorized output.
         """
-        class DryBackend:
-            __getattr__ = lambda self, attr: (lambda *args: None)
-
+        backend = self._DryBackend()
```

Comment:
Why introduce `backend` here? It is used only once in line 181. Why not directly use `self._DryBackend()` in line 181, similar to lines 596 and 600?

Reply:
a41de1b6dafcbcb9d00bbc8d2cb1d82cd54b8df7